core/src/test/java/tech/tablesaw/io/DataFrameReaderTest.java

```diff
@@ -50,8 +50,7 @@ public void csv() throws IOException {
   @Test
   public void readUrlWithExtension() throws Exception {
     URL url =
-        mockUrlHelper(
-            "http://something.other.com/file.csv", ImmutableList.of("region", "canada", "us"));
+        mockUrlHelper("something.other.com/file.csv", ImmutableList.of("region", "canada", "us"));
```

Comment:
It looks weird to me that `http://` is missing. I didn't understand the explanation for why it had to be changed. Is there a fix we can make to `mockUrlHelper` or something that will allow `http://` to be used?

Reply:
Changed the mockUrl to be able to use urls